packages/components/psammead-grid/README.md

```diff
@@ -55,56 +78,53 @@ When we refer to `group3` in this component, we're referring to the breakpoint w
 
 Here is an example of a `<Grid>` that has 8 columns for `group4` (from 1008px to 1279px). It has two child `<Grid item>` elements, one which spans 6/8 columns and one which spans 2/8 columns within this breakpoint.
 
-In this example groups 1, 2, 3, & 5 are omitted from the `columns` prop - so at those breakpoints items fill the whole width of the page.
-
 ```jsx
 import Grid from '@bbc/psammead-grid';
 
 const MyComponent = () => (
-  <Grid columns={{ group4: 8 }}>
-    <Grid item columns={{ group4: 6 }}>
+  <Grid
+    columns={{
+      group0: 8,
+      group1: 8,
+      group2: 8,
+      group3: 8,
+      group4: 8,
+      group5: 8,
+    }}
+  >
+    <Grid
+      item
+      columns={{
+        group0: 6,
+        group1: 6,
+        group2: 6,
+        group3: 6,
+        group4: 6,
+        group5: 6,
+      }}
+    >
       <p>Item 1 - Paragraph that spans 6 out of 8 columns through group4</p>
     </Grid>
-    <Grid item columns={{ group4: 2 }}>
+    <Grid
+      item
+      columns={{
+        group0: 2,
+        group1: 2,
+        group2: 2,
+        group3: 2,
+        group4: 2,
+        group5: 2,
+      }}
+    >
       <p>Item 2 - Paragraph that spans 2 out of 8 columns through group4</p>
     </Grid>
-    <Grid item columns={{ group4: 2 }}>
-      <p>Item 3 - Paragraph that spans 2 out of 8 columns through group4</p>
-    </Grid>
-    <Grid item columns={{ group4: 2 }}>
-      <p>Item 4 - Paragraph that spans 2 out of 8 columns through group4</p>
-    </Grid>
   </Grid>
 );
 ```
 
 Image of this example when viewed at within `group4` (from 1008px to 1279px). When you look at this example at any other breakpoint, each of the four items fill up the full width of the row.
 ![8 column grid with items 6 2 2 2 with labels](packages/components/psammead-grid/documentation/8-column-grid-with-items-6-2-2-2-with-labels.png)
 
-#### <a name="multiple-groups">Setting values for multiple groups</a>
```

Comment:
Why has this example been removed?

Reply:
All grid examples in this documentation now have multiple column groups, this example is no longer needed.